README.md

```diff
@@ -41,6 +41,19 @@ brew install awegrzyn/influxdata/influxdb-cxx
 
 ## Quick start
 
+### Include in CMake project
+
+The InfluxDB library is exported as target `InfluxData::InfluxDB`.
+
+```cmake
+project(test)
+
+find_package(InfluxDB)
+
+add_executable(test main)
```

Comment:
Using `test` might be problematic as it conflicts with the cmake internal target name (if tests are enabled). Maybe just use "example" instead. Also for completeness the source file should include `.cpp`, so it's a ready to use example.

Reply:
Done. The `.cpp` is not required and CMake also does not recommend to add the extension. But I'm unfortunately I don't remember where I read that. But maybe it makes things more clear in the example, so updated to your version.

Reply:
Didn't know that, thanks for the info! :smile: